cg/services/order_validation_service/workflows/mip_rna/validation_rules.py

```diff
@@ -0,0 +1,50 @@
+from cg.services.order_validation_service.rules.case.rules import (
+    validate_case_internal_ids_exist,
+    validate_case_names_available,
+    validate_case_names_not_repeated,
+)
+from cg.services.order_validation_service.rules.case_sample.rules import (
+    validate_application_compatibility,
+    validate_application_exists,
+    validate_application_not_archived,
+    validate_buffer_skip_rc_condition,
+    validate_concentration_interval_if_skip_rc,
+    validate_concentration_required_if_skip_rc,
+    validate_container_name_required,
+    validate_required_volume,
+    validate_sample_names_not_repeated,
+    validate_samples_exist,
+    validate_subject_ids_different_from_case_names,
+    validate_subject_ids_different_from_sample_names,
+    validate_tube_container_name_unique,
+    validate_volume_interval,
+    validate_well_position_format,
+    validate_well_positions_required,
+    validate_wells_contain_at_most_one_sample,
+)
+
+CASE_RULES: list[callable] = [
+    validate_case_internal_ids_exist,
+    validate_case_names_available,
+    validate_case_names_not_repeated,
+]
+
+CASE_SAMPLE_RULES: list[callable] = [
```

Comment:
We seem to have missed implementing this rule 

<img width="717" alt="image" src="https://github.com/user-attachments/assets/992c439e-79a8-48cc-98d8-436c40cba9ab">

Seems to have been missed for other workflows as well so it might warrant its own PR where we add it to all relevant order types in one go?

Reply:
It seems to exist for Balsamic so we just need to widen its scope!

Reply:
I can see that it is implemented 🙌 Could you make this list in alphabetical order so that it is easier to read?